include/help_util.h

```diff
@@ -0,0 +1,29 @@
+#ifndef HELP_UTIL_H
+#define HELP_UTIL_H
+
+#include <map>
+#include <string>
+
+namespace HelpUtil {
+
+enum class Filter { ALL, COMMON };
+
+enum class Category { MISC, FILE, DOSBOX, BATCH };
+
+enum class CmdType { SHELL, PROGRAM };
+
+struct Detail {
+	Filter filter{Filter::ALL};
+	Category category{Category::MISC};
+    CmdType type{CmdType::SHELL};
```

Comment:
The code in general doesn't use { } constructors.
Suggest:
``` c++
struct Detail {
	Filter filter = Filter::ALL;
	Category category = Category::MISC;
	CmdType type = CmdType::SHELL;
	std::string name = {};
};
```

A bit easier on the eyes, I find, too.